### ISSUE TITLE
Scroll sensitivity  

### ISSUE THREAD

**Issue by criesner** [hakimel/reveal.js#817]
With my might mouse when I scroll down the slide jumps 2-3 slides ahead. Is there any settings to control how sensitive the scroll is, I would like to make it less touchy. 

Thanks!

**Comment by tschann** [comment]
Yeah I'm having the same issue. Macbook pro touchpad...goes 2-3 slides on just a slight two-finger drag.

**Comment by ilyaglar** [comment]
Got the same issues with macbooks

**Comment by shikolay** [comment]
👍 to make that constant(600) a config

**Comment by kerr325** [comment]
Just to add a little clarity to @ilyaglar post above this setting can be found on line ~4586 in reveal.js where the default code is 

`if( Date.now() - lastMouseWheelStep > 600 ) {`

I found adjusting it to 1400 was a sweet spot

> if( Date.now() - lastMouseWheelStep > 1400 ) {
